alienfile

```diff
@@ -53,12 +53,6 @@ share {
 
     plugin Extract => 'zip';
 
-    patch sub {
-        # README is a symlink and would make autotools fail on filesystems not supporting them
-        unlink 'README';
-        copy qw(README.md README);
```

Comment:
If this works now we can also remove the `File::Copy` import since I think this is the only place it is used.

Reply:
Nice catch! 😃 I updated